test/evp_xof_test.c

```diff
@@ -257,7 +257,10 @@ static int shake_kat_digestfinal_xoflen_test(void)
     params[0] = OSSL_PARAM_construct_size_t(OSSL_DIGEST_PARAM_XOFLEN, &sz);
     params[1] = OSSL_PARAM_construct_end();
 
-    if (!TEST_int_eq(EVP_MD_CTX_set_params(ctx, params), 1)
+    if (!TEST_int_eq(EVP_MD_CTX_size(ctx), -1)
+        || !TEST_int_eq(EVP_MD_CTX_set_params(ctx, params), 1)
```

Comment:
Could you please add an explicit EVP_MD_xof() test call here so we have a unit test?

Reply:
added positive and negative tests